src/org/zaproxy/zap/extension/api/CoreAPI.java

```diff
@@ -1276,6 +1302,25 @@ public void process(HttpMessage msg) {
 		}
 	}
 
+	private static List<Integer> getIds(JSONObject params) throws ApiException {
+		List<Integer> listIds = new ArrayList<>();
+		for (String id : params.getString(PARAM_IDS).split(",")) {
+			String trimmedId = id.trim();
```

Comment:
Unfortunately that's not possible, the split returns an array. I could remove the empty check and just call trim when invoking the parse?

Or, we could use streams:
```Java
listIds = Arrays.stream(params.getString(PARAM_IDS).split(","))
        .map(String::trim)
        .filter(e -> !e.isEmpty())
        .map(Integer::valueOf)
        .collect(Collectors.toList());
```